zebra/zebra_ns.h

```diff
@@ -49,6 +49,8 @@ struct zebra_ns {
 	struct nlsock netlink_dplane_out;
 	struct nlsock netlink_dplane_in;
 	struct event *t_netlink;
+
+	struct nlsock ge_netlink_cmd; /* command channel for generic netlink */
```

Comment:
let's just use the zebra_router_get_next_sequence.  It will stay consistent with everything else we do.  FRR uses one sequence number generator for all it's netlink sockets.